CONTRIBUTING.md

```diff
@@ -58,6 +58,44 @@ module.exports = function ModuleName(options,UI) {
 }
 ```
 
+Modules can be distributed independently via npm, yarn and cdn(for browser). `require` syntax should not be used if the module is intended to be used inside the browser.
+Any Independent Module Must follow this basic format
+
+```js
+function ModuleName(options,UI) {
+
+  var output;
+
+  function draw(input,callback) {
+
+    var output = function(input){
+      /* do something with the input */
+      return input;
+    }
+
+    this.output = output(input); // run the output and assign it to this.output
+    callback();
+  }
+
+  return {
+    options: options,
+    draw: draw,
+    output: output,
+    UI: UI
+  };
+}
+/* Info can be defined here or imported from a json file */
+var info = {
```

Comment:
Here since we now only have one means of reporting `info`, i think we could add it to the `module.exports` line below directly, as I showed in the suggestion above. It'll be a little bit cleaner!

Reply:
@jywarren but for a node only module this may be imported from a json file, so then it'll be like 
```js
var info = require('./info')
```

Reply:
Ah, but couldn't it be like:
```js


module.exports = [
  ModuleName,
  require('./info')
];
```

?

Reply:
Oh yeah I can do that